src/training/ui/menu.rs

```diff
@@ -172,6 +189,7 @@ unsafe fn render_toggle_page(app: &App, root_pane: &mut Pane) {
                     title_bg_material.set_black_res_color(BG_LEFT_OFF_BLACK_COLOR);
                 }
 
+                // Replace with setting the check mark to visible
```

Comment:
Should this have a `TODO:` in the comment?

Reply:
Yes, it should! I'll add that. Good catch!

Reply:
I was intending to do that before the PR was made.